packages/shared/getComponentName.js

```diff
@@ -90,7 +90,7 @@ function getComponentName(type: mixed): string | null {
         if (resolvedThenable) {
           return getComponentName(resolvedThenable);
         }
-        break;
+        return 'Lazy';
```

Comment:
I think to fully support the `displayName` attribute you describe, we'd want to do this:
```js
case REACT_LAZY_TYPE: {
  const thenable: LazyComponent<mixed> = (type: any);
  const resolvedThenable = refineResolvedLazyComponent(thenable);
  if (resolvedThenable) {
    return getComponentName(resolvedThenable);
  } else {
    return thenable.displayName || 'Lazy';
  }
}
```